lib/SILGen/SILGenDestructor.cpp

```diff
@@ -247,13 +389,28 @@ void SILGenFunction::emitClassMemberDestruction(ManagedValue selfValue,
                                                finishBB);
   }
 
+  // Before we destroy all fields, we check if any of them are
+  // recursively the same type as `self`, so we can iteratively
+  // deinitialize them, to prevent deep recursion and potential
+  // stack overflows.
+  llvm::SmallSetVector<VarDecl*, 4> recursiveLinks;
```

Comment:
Another instance showing git-clang-format missing. Can you git-clang-format this?

Reply:
`git-clang-format` appears to accept this as is, as well as with added whitespace. It also accepts `Type* variable;` and `Type *variable;`. I'm happy to change this manually, but maybe the formatting rules have to be adjusted?